assets/js/components/notifications/BannerNotifications.js

```diff
@@ -56,6 +56,9 @@ export default function BannerNotifications() {
 	const adSenseModuleActive = useSelect( ( select ) =>
 		select( CORE_MODULES ).isModuleActive( 'adsense' )
 	);
+	const twGModuleAvailable = useSelect( ( select ) =>
+		select( CORE_MODULES ).isModuleAvailable( 'thank-with-google' )
```

Comment:
The value here should be the same as the state of the feature flag, unless someone of course filtered the module out. In that case, `isModuleAvailable` should be used in place of the `twg` feature flag for module-availability checks.